### DISCUSSION TITLE
Is it possible in Nuxt 3 to create a fully statically generated site that runs in the client only without a server?

### DISCUSSION THREAD

**Discussion by andyjamesn** [nuxt/nuxt#26006] [General]
I have been trying to get this working. I have a bare bones Nuxt App and running `npx nuxi generate` with `ssr: true`

This generates the output folder. I can manally upload this to Netlify and it works fine. 

However if I try run the output/index.html locally with VS Code Live Server or Upload it to a static web host I get 404 errors on all the javascript files

![CleanShot 2024-02-29 at 00 15 58@2x](https://github.com/nuxt/nuxt/assets/54320410/22be8d34-24b2-41da-b3cb-f88fd542c577)

I thought if it was statically generated it would work without the need for a server and just work in the client. So I am trying to understand what I am missing.

**Comment by TheAlexLichter** [comment]
You need to upload the whole output folder as JS and CSS (also static assets) are needed too!

**Reply by andyjamesn** [reply]
This is what I am doing. Well the public folder inside output is being uploaded. So 

.output/public. The only other file in .output is nitro.json.

This is a fresh nuxt install and all I did was change the App to use <NuxtPage /> Then added a pages/index.vue with basic .vue file with Hello world text, not script code, no assets. 

I then run npx nuxi generate and upload the .output folder and get these 404 errors.

P.S. Your youtube videos are amazing!

**Comment by andyjamesn** [comment]
I got it working. I had to rename the `output/public/_nuxt` folder using `app.buildAssetsDir: 'nuxt'` (for some reason our server does not like the _nuxt folder) and then I moved the **nuxt** folder to the top level directory of our server.
